activerecord/test/cases/associations/inverse_associations_test.rb

```diff
@@ -873,6 +873,25 @@ def test_recursive_inverse_on_recursive_model_has_many_inversing
     end
   end
 
+  def test_should_raise_error_for_non_bidirectional_inverses
+    face_klass = Class.new(ActiveRecord::Base) do
+      self.table_name = Face.table_name
+
+      def self.name
+        "Face"
+      end
+    end
+
+    # Correct inverse_of should be `:face`, but the incorrect configuration `:interests` was used.
+    face_klass.belongs_to(:human, inverse_of: :interests)
+
+    error = assert_raises(ActiveRecord::InverseOfAssociationNotBidirectionalError) do
```

Comment:
You can use [assert_raises(match:)](https://edgeapi.rubyonrails.org/classes/ActiveSupport/Testing/Assertions.html#method-i-assert_raises).

```
assert_raises(AR::IoANotFoundError, match: /Inverse association human (:association in Huma) does not exist./)
```